module/extension/Director/tests/entire_task.cpp

```diff
@@ -0,0 +1,89 @@
+#include <catch.hpp>
+#include <nuclear>
+
+#include "Director.hpp"
+#include "TestBase.hpp"
+
+// Anonymous namespace to avoid name collisions
+namespace {
+
+    template <int i>
+    struct SimpleTask {
+        std::string msg;
+    };
+    struct ComplexTask {
+        std::string msg;
+    };
```

Comment:
In the other files you have a constructor for these structs :thinking:

Reply:
Thought I had normalised a constructor on all of them, will fix